docs/configuration/parsing-methods.rst

```diff
@@ -0,0 +1,57 @@
+.. _parsing-methods:
+
+Parsing Methods
```

Comment:
@jlaneve would it make sense for us to rename from `load_mode` to `parse_method` and `LoadMode` to `ParseMethod`, so we are consistent with the terminology used in the docs and code?

Reply:
we talked about this - going to leave this as-is in the code and just document it as parsing methods